pkg/helpers/helpers.go

```
@@ -28,22 +32,20 @@ func GetEnv(key, fallback string) string {
 	return fallback
 }
 
+// GetAccessToken returns a Github OAuth Token
 func GetAccessToken(config config.Config, installationID int64) (string, error) {
-	token := os.Getenv("personal_access_token")
+	token := os.Getenv("PERSONAL_ACCESS_TOKEN")
```

Not necessarily for this PR, but it's better to load these values into a config object early, and providing via flags, and fail fast, as it makes debugging and running more stable, rather than failing later during command

---

bit of an issue with how I'm hosting, but I'll look into it, I know the DO has the commands that you can inject, might be able to do